extensions/sql-database-projects/src/common/constants.ts

```diff
@@ -165,7 +165,7 @@ export function unableToPerformAction(action: string, uri: string) { return loca
 export function unableToFindObject(path: string, objType: string) { return localize('unableToFindFile', "Unable to find {1} with path '{0}'", path, objType); }
 export function deployScriptExists(scriptType: string) { return localize('deployScriptExists', "A {0} script already exists. The new script will not be included in build.", scriptType); }
 export function notValidVariableName(name: string) { return localize('notValidVariableName', "The variable name '{0}' is not valid.", name); }
-export function cantAddCircularProjectReference(project: string) { return localize('cantAddCircularProjectReference', "A reference to project '{0} cannot be added. Adding this project as a reference would cause a circular dependency", project); }
+export function cantAddCircularProjectReference(project: string) { return localize('cantAddCircularProjectReference', "A reference to project {0} cannot be added. Adding this project as a reference would cause a circular dependency", project); }
```

Comment:
I think I lean the other direction: adding the missing single quote so it's obvious that's the referenced object and so that it still makes sense for any project name.